pages/about.html

```diff
@@ -4,9 +4,44 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/Users/marilynthai/Developer/projects/personal-portfolio-site/styles/style.css" rel="StyleSheet">
```

Comment:
Careful with your links! This is looking for a specific file on your computer and will not work on any other place. Instead, you should use a relative path. In this case, it would be `../styles/style.css`. The two dots indicate to start at the parent of the current directory.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/

This applies for image links as well.